source/_integrations/mqtt.markdown

```diff
@@ -334,6 +329,13 @@ Documentation on the MQTT components that support MQTT discovery [can be found h
 
 ### Discovery messages
 
+Home Assistant supports two types of discovery messages:
+
+- [Device-based discovery](/integrations/mqtt/#device-based-discovery-payload), which allows you to include several components in a single discovery message
+- [Single component discovery](/integrations/mqtt/#single-component-discovery-payload), where you publish a separate discovery message for each component
```

Comment:
I'd rather like to use the term "Device discovery" as it meant to discover an MQTT device.
The Single component discovery can setup a component, but the device part is not required.